Makefile-docker

```diff
@@ -28,12 +28,12 @@ check_debian_packages: ## check the existence of multiple debian packages
 
 .PHONY: check_pip_packages
 check_pip_packages: ## check the existence of multiple python packages
-	@ ./scripts/check_pip_packages.sh prod.txt
-# "production" corresponds to the "propduction" DOCKER_TARGET defined in the Dockerfile
-# When the target is "production" it means we cannot expect dev.txt dependencies to be installed.
-	@if [ "$(DOCKER_TARGET)" != "production" ]; then \
-		./scripts/check_pip_packages.sh dev.txt; \
-	fi
+	./scripts/check_pip_packages.sh pip.txt
+	./scripts/check_pip_packages.sh prod.txt
+# We expect dev dependencies only if DOCKER_TARGET is not production
+ifeq ($(DOCKER_TARGET), development)
+	./scripts/check_pip_packages.sh dev.txt
+endif
```

Comment:
my Makefile foo is non-master level, but should these lines be indented like they were before?

Reply:
The if statements don't have to be indendted and common make examples show these statements non indented. Make is quite strange in parsing but this works and I'd keep it this way.